src/libraries/System.Diagnostics.TraceSource/tests/System.Diagnostics.TraceSource.Config.Tests/TraceSourceWithConfigurationTests.cs

```diff
@@ -35,7 +36,7 @@ private static void CreateAndLoadConfigFile(string filename)
 
         [Fact]
         [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/74244", TestPlatforms.tvOS)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/74244", TestPlatforms.tvOS | TestPlatforms.iOS | TestPlatforms.Android)]
```

Comment:
Does this mean that the test is also failing with Mono?

Reply:
Yes, they were disabled on platforms which don't have assemblies.